clusterman/batch/node_migration.py

```diff
@@ -53,36 +58,97 @@ def configure_initial(self):
         self.sqs_client = sqs()
         self.migration_workers = {}
         self.migration_configs = {}
+        self.events_in_progress = set()
+        self.pools_accepting_events = set()
+        self.event_queue = staticconf.read_string(f"clusters.{self.options.cluster}.migration_event_queue_url")
         self.run_interval = staticconf.read_int("batches.node_migration.run_interval_seconds", 60)
```

Comment:
not: We will use this `service` for choosing victim node. Probably we need to check again when new node come up. Long story short default value may be `average node creation time` which is 5 mins IIRC

Reply:
node monitoring will be done by workers, the main loop here will just check that migration worker processes are healthy.